Library/Formula/scriptcs.rb

```diff
@@ -0,0 +1,25 @@
+require "formula"
+
+class Scriptcs < Formula
+  homepage "https://github.com/scriptcs/scriptcs"
+  url "https://github.com/scriptcs/scriptcs.git", :tag => "v0.13.3"
```

Comment:
Are there submodules involved? A tarball is preferred if not. If there are submodules involved, please add a ` :revision => "blah" ` in addition to the ` :tag `. The ` :revision ` points to the upstream commit the ` :tag ` is based on.